packages/fiori-freestyle-writer/test/__snapshots__/basic.test.ts.snap

```diff
@@ -164,7 +164,7 @@ title=App Title",
     </style>
     <script
         id=\\"sap-ui-bootstrap\\"
-        src=\\"resources/sap-ui-core.js\\"
+        src=\\"https://ui5.sap.com/1.78.11/resources/sap-ui-core.js\\"
```

Comment:
I'm wondering how this will affect projects deployed to CF? If its hardcode, then they'll have to change the files in source control and redeploy.
This is why the destination is used in CF deployment i.e. 

https://github.com/SAP-samples/fiori-tools-samples/blob/main/cap/cap-fiori-mta/app/feproject-ui/xs-app.json#L15

https://github.com/SAP-samples/fiori-tools-samples/blob/main/cap/cap-fiori-mta/mta.yaml#L100

The advantage of this is, the developer can login to their subaccount, select the service instance and change the UI5 URL to point any version thus not requiring a rebuild and redeploy.

Reply:
This is how CAP has been configured in Fiori tools to support loading during local preview as no proxy was provided by `cds-serve`. This has changed e.g. for typescript configurations using https://www.npmjs.com/package/cds-plugin-ui5. But a general change to this behaviour is out of scope of the current TBI as it has wider implications to be considered in a separate TBI.